src/Firebase/Exception/AuthApiExceptionConverter.php

```diff
@@ -121,6 +123,22 @@ private function convertGuzzleRequestException(RequestException $e): AuthExcepti
             return new ProviderLinkFailed('This credential is already associated with a different user account.', $code, $e);
         }
 
+        if (\mb_stripos($message, 'USER_DISABLED') !== false) {
+            return new UserDisabled('The user account has been disabled by an administrator.', $code, $e);
+        }
+
+        if (\mb_stripos($message, 'OPERATION_NOT_ALLOWED') !== false) {
+            return new OperationNotAllowed('Operation not allowed.', $code, $e);
+        }
+
+        if (\mb_stripos($message, 'EXPIRED_OOB_CODE') !== false) {
```

Comment:
```suggestion
        if (\mb_stripos($message, 'expired_oob_code') !== false) {
```

`mb_stripos()` is case-insensitive and all other checks work with lowercased string. I implemented the checks in this way to be prepared just in case™ 😅